connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorsResource.java

```diff
@@ -173,6 +176,24 @@ public ConnectorStateInfo getConnectorStatus(final @PathParam("connector") Strin
         return herder.connectorStatus(connector);
     }
 
+    @GET
+    @Path("/{connector}/topics")
+    public Map<String, ActiveTopicsInfo> getConnectorActiveTopics(final @PathParam("connector") String connector) throws Throwable {
+        return herder.connectorActiveTopics(connector);
```

Comment:
Should this method return an error if the worker is configured to not enable the active topic feature?

Reply:
Many of the other methods don't involve an optional feature. It may be possible to change the return type to `Response`, and then when the feature is disabled return a response like in `resetConnectorActiveTopics(...)` but then when the feature is enabled build the map to be returned and then:
```
return Response.ok(map).build();
```

Reply:
Good point! 
Changed the return type to `Response` to return an error as well.